source/_redirects

```diff
@@ -2349,4 +2346,3 @@
 /components/sensor.iota /more-info/removed-integration 301
 /components/iota /more-info/removed-integration 301
 /integrations/iota /more-info/removed-integration 301
```

Comment:
We need to redirect the removed pages, by adding these to the end of the redirect file:

```
/components/mychevy /more-info/removed-integration 301
/integrations/mychevy /more-info/removed-integration 301
/components/binary_sensor.mychevy /more-info/removed-integration 301
/components/sensor.mychevy /more-info/removed-integration 301
```

Reply:
Done